docs/main/ios/spm.md

```diff
@@ -59,15 +59,79 @@ Then let's sync the web app. This will add the App plugin SPM to the iOS project
 
 You can now use the App plugin normally.
 
-<em>More details coming soon</em>
+## Using SPM in an existing Capacitor Project
+
+First, ensure you have a backup of the current state of your project, either in source control or elsewhere.
+
+### Deleting your iOS Directory
+
+If you **have not manually changed your Xcode Project at all**, one option to migrate is to delete the `ios` directory and then run `npx cap add ios --packagemanager SPM`. This will remove the CocoaPods template project and replace it with the SPM template project.  
+### Using our migration tool
+
+The Capacitor CLI has a command to help migrate from CocoaPods to Swift Package Manager. However, one manual step is still required. In addition, projects with Cordova plugins will not be migrated correctly and neither will projects that use plugins that do not have SPM versions available.
+
+To start, run `npx cap spm-migration-assistant` in the root of your project.
+
+This tool will:
+  - Run `pod deintegrate` removing CocoaPods
+  - Delete the `Podfile`, `App.xcworkspace`, and `Podfile.lock`
+  - Create a `CapApp-SPM` directory with the needed files
+  - Generate a `Package.swift` from your plugins, and warn you if any can't included. 
+
+After this is run, run a `npx cap sync` again.
+
+Then run `npx cap open ios` and you should see something similar to this:
+
+![Migrate Step 1](../../../static/img/spm/xcode-step-1.png)
+
+Highlight App, and Select the Package Dependencies tab, and on this page press the + symbol to add a dependency:
+
+![Migrate Step 2](../../../static/img/spm/xcode-step-2.png)
+
+You should see something similar to the below - select Add Local... from the dialog:
+
+![Migrate Step 3](../../../static/img/spm/xcode-step-3.png)
+
+Select CapApp-SPM in this dialog and click Add Package:
+
+![Migrate Step 4](../../../static/img/spm/xcode-step-4.png)
+
+Click Add Package again when this screen shows up: 
+
+![Migrate Step 5](../../../static/img/spm/xcode-step-5.png)
+
+When you are done, you should see a screen like this. At this point you're done and can build and work as normal:
```

Comment:
Here it says "you're done", but we still have the `debug.xcconfig` step after this, so that step should be before or this text should be changed.